### PR TITLE
DOCS-1959: Edit mousemover tutorial for R2D2

### PR DESCRIPTION
**Pull request by sguequierre**
* Update tutorial for R2D2
* Update servo control tab video

### CODE REVIEW

File: docs/tutorials/get-started/servo-mousemover.md [hunk 1]
```diff
@@ -27,12 +27,19 @@ Maybe it was to clear your head, grab your lunch, answer the door, and you get i
 {{<youtube embed_url="https://www.youtube-nocookie.com/embed/JVtZ7aU4k68">}}
 
 This tutorial will show you how to build a mouse mover using Viam, a Raspberry Pi, a servo, and an optical mouse.
-This robot will turn the continuous servo that's secured inside the box, which will turn the circle under the optical mouse. This will keep your computer from falling asleep.
+This robot will turn the continuous servo that's secured inside the box, which will turn the circle under the optical mouse.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:30]:
```suggestion
This machine will turn the continuous servo that's secured inside the box, which will turn the circle under the optical mouse.
```

File: docs/tutorials/get-started/servo-mousemover.md [hunk 2]
```diff
@@ -27,12 +27,19 @@ Maybe it was to clear your head, grab your lunch, answer the door, and you get i
...
+This will keep your computer from falling asleep.
 
 This project is a good place to begin if you're new to robotics and would like to learn how to use a [servo component](/components/servo/) with the Viam app and Viam's [Python SDK](https://python.viam.dev/).
 
-<div class="td-max-width-on-larger-screens">
-  {{<gif webm_src="/tutorials/single-component-tutorials-servo-mousemover/moving.webm" mp4_src="/tutorials/single-component-tutorials-servo-mousemover/moving.mp4" alt="Video of a mouse running erratically on the screen in a sweeping motion in front of the Viam app Control page with an overlaid video of a mouse on top of a cardboard box with a moving cardboard circle underneath it with red swirl lines on the circle.">}}
+<div class="row">
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:35]: Nice - breaks a bit on mobile. I'll file a ticket for it.

File: docs/tutorials/get-started/servo-mousemover.md [hunk 3]
```diff
@@ -64,7 +71,8 @@ This project is a good place to begin if you're new to robotics and would like t
 
 ## Install Viam software
 
-First, install `viam-server` according to our [installation guide](/get-started/installation/)
+First, prepare your Raspberry Pi according to our [setup guide](/get-started/installation/prepare/rpi-setup/).
+Follow the instructions to install the right OS, connect with SSH, add a new machine in the [the Viam app](https://app.viam.com), and install `viam-server`: the software that you will use to operate your machine.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:75]:
```suggestion
Follow the instructions to install the right OS and connect with SSH.
Then add a new machine in the [the Viam app](https://app.viam.com) and follow the setup instructions to install `viam-server` on your Raspberry Pi and connect to the Viam app.
```

File: docs/tutorials/get-started/servo-mousemover.md [hunk 4]
```diff
@@ -80,29 +88,28 @@ To verify and get the version of the package, you can run the command:
...
 
-Since we already installed the Python SDK, we can skip the first step.
-Copy the boilerplate Python code under the second step.
-
-![A screenshot of the Viam app showing the Code sample tab. On this tab it has Language: with four boxes, Python is a black background with white text and has a checkmark indicating it is selected, then there's Golang, Typescript (Web), and Remotes with white backgrounds and black text. The page next says Python SDK (Documentation) with a link to Documentation. Below that there's a place that says "pip install viam-SDK" this is how you install the Viam Python SDK. Below that instruction are the instructions to "Then copy and run the following code to connect to your robot:" with 7 lines of code seen (the screenshot is incomplete).](/tutorials/single-component-tutorials-servo-mousemover/copy-python-code-sample.png)
+Since you already installed the Python SDK, skip the first step.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:104]: nice

File: docs/tutorials/get-started/servo-mousemover.md [hunk 5]
```diff
@@ -153,58 +160,51 @@ Once you have the wires connected, attach the wheel/arm to servo and turn on the
...
-
-Click **Create component** in the lower-left corner.
-
-Select `board` for type and `pi` for model.
-
-Enter `local` as the name for your board, then click **Create**.
+- Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+- Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+- Select the `board` type, then select the `pi` model.
+- Enter the name `local` for your board and click **Create**.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:172]:
```suggestion
1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
1. Select the `board` type, then select the `pi` model.
1. Enter the name `local` for your board and click **Create**.
```
when you have to do steps in a specific order, use an ordered list.

File: docs/tutorials/get-started/servo-mousemover.md [hunk 6]
```diff
@@ -153,58 +160,51 @@ Once you have the wires connected, attach the wheel/arm to servo and turn on the
...
 
 ### Servo component
 
 Create a [servo component](/components/servo/):
 
-Click **Create component** in the lower-left corner.
+- Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+- Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+- Select the `servo` type, then select the `pi` model.
+- Enter the name `fs90f` for your servo and click **Create**.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:185]:
```suggestion
1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
1. Select the `servo` type, then select the `pi` model.
1. Enter the name `fs90f` for your servo and click **Create**.
```

**Review by npentrel — APPROVED** [review]
LGTM % nits

File: docs/tutorials/get-started/servo-mousemover.md (hunk 2 above)
Comment (sguequierre) [reply docs/tutorials/get-started/servo-mousemover.md:35]: Thank you!

**Pull request merged**